packages/client/lib/miner/miner.ts

```diff
@@ -126,47 +154,52 @@ export class Miner {
     } else {
       // Set the state root to ensure the resulting state
       // is based on the parent block's state
-      await vmCopy.stateManager.setStateRoot(parentBlock.header.stateRoot)
+      await vmCopy.stateManager.setStateRoot(parentBlockHeader.stateRoot)
     }
 
     let difficulty
     if (this.config.chainCommon.consensusType() === ConsensusType.ProofOfAuthority) {
       // Determine if signer is INTURN (2) or NOTURN (1)
-      const inTurn = vmCopy.blockchain.cliqueSignerInTurn(
-        signerAddress,
-        parentBlock.header.number.addn(1)
-      )
+      const inTurn = vmCopy.blockchain.cliqueSignerInTurn(signerAddress, number)
       difficulty = inTurn ? 2 : 1
     }
 
+    let baseFeePerGas
+    const londonHardforkBlock = this.config.chainCommon.hardforkBlockBN('london')
+    const isInitialEIP1559Block = londonHardforkBlock && number.eq(londonHardforkBlock)
+    if (isInitialEIP1559Block) {
```

Comment:
Ok, we can also just stay on the status quo for now since it's not making anything structurally worse. 🙂

Will likely be worth to keep an eye on this though.